src/components/MessagesList/MessagesGroup/MessagesSystemGroup.vue

```diff
@@ -133,28 +133,27 @@ export default {
 				return '' // No previous message
 			}
 
+			if (message1.actorId !== message2.actorId
+				|| message1.actorType !== message2.actorType) {
+				return '' // Different actors
```

Comment:
this breaks the grouping of `A, B, C joined the call`

Please add tests for those cases